docs/operators/http_input.md

```diff
@@ -0,0 +1,127 @@
+## `http_input` operator
+
+The `http_input` operator will send entries to Google Cloud Logging.
```

Comment:
We should update this description. This makes it sound like the operator's responsibility is sending things to Google Cloud Logging.